app/assets/javascripts/components/admin/unmatched_page.es6.jsx

```diff
@@ -27,11 +28,13 @@ class UnmatchedPage extends React.Component {
     );
   }
 
-  studentOnClick(studentId) {
+  studentOnClick(studentId, instrument) {
     var studentRoute = ApiConstants.students.show(studentId)
     var studentResolve = ((response) => {
-      this.setState({fullStudent: true, student: response});
-      var teacherRoute = ApiConstants.teachers.possibleTeachers(studentId);
+      this.setState({fullStudent: true, 
```

Comment:
put first attribute in new line 
so 
this.setState({
    fullStudent: true,
   });